flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/testutils/executor/TestExecutorExtension.java

```diff
@@ -49,6 +58,20 @@ public T getExecutor() {
     public void afterAll(ExtensionContext context) throws Exception {
         if (executorService != null) {
             executorService.shutdown();
+
+            try {
```

Comment:
Is this change related?

Reply:
No, this change is not related. I was too lazy to create a Jira issue for that one and just added it as a minor hotfix commit. But you're right. It's too different to let it slip through. :-) I created FLINK-32039 to cover it.